Assets/_MUTUAL/Screens/LoadCaseScreen.cs

```diff
@@ -0,0 +1,165 @@
+﻿using Assets.CaseFile;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Mutual.Screens
+{
+    class LoadCaseScreen : ILoadCaseScreen
+    {
+        #region Dependencies
+
+        private readonly Project m_Project;
+
+        #endregion
+
+        #region Private Constants
+
+        private const string ParentTag = "UIParent";
+        private const string LoadCaseBTNName = "LoadCaseBTN";
+        private const string ViewBTNName = "ViewBTN";
+
+        #endregion
+
+        #region Private Members
+
+        private Button m_LoadCaseBtn;
+        private Button m_ViewBtn;
+        private Transform m_Parent;
+        private List<GameObject> m_Meshes;
+
+        #endregion
+
+        #region Constructors
+
+        public LoadCaseScreen(Project project)
```

Comment:
@RemyaTP  Revisit after Project/Patient restructure

Understand that this may simply be temporary - but I don't really think we should be loading a screen via a specific Project (or patient) object.  Ideally the Screen is loaded, and the screen itself seeks a reference to a patient/project object (using Zenject, or a reference globally accessibly Manager class?)  We'll never be loading more than one patient / project at one time.

Reply:
Are we expecting to create a manager class now?

Reply:
This would be a good way of solving it in the future, unrequired for this PR.